.storybook/style.scss

```diff
@@ -0,0 +1,52 @@
+@import url( 'https://wordpress.org/wp-admin/load-styles.php?c=0&dir=ltr&load%5B%5D=common,buttons,dashicons,forms' );
+
+html {
```

Comment:
I came up with these styles to match the `info` addon. The infoAddon automatically applies some styling to its output, and for the stories that don't use it, I came up with the styles to unify the output